opensrp-anc/src/test/java/org/smartregister/anc/library/interactor/ContactInteractorTest.java

```diff
@@ -142,4 +170,76 @@ public void testFinalizeContactFormInvokesUpdatesPatientRepositoryWithCorrectPar
+            partialContact.setFinalized(false);
+            partialContact.setSortOrder(1);
+            partialContact.setType("anc_quick_check");
+            partialContactList.add(partialContact);
+
+
+            PowerMockito.mockStatic(AncLibrary.class);
+            PowerMockito.mockStatic(PatientRepositoryHelper.class);
+            PowerMockito.mockStatic(PreviousContactRepositoryHelper.class);
+            PowerMockito.mockStatic(PartialContactRepositoryHelper.class);
+            PowerMockito.mockStatic(EventClientRepository.class);
+            PowerMockito.mockStatic(LocationHelper.class);
+            PowerMockito.mockStatic(Pair.class);
+
+
+            PowerMockito.when(PatientRepositoryHelper.getWomanProfileDetails(DUMMY_BASE_ENTITY_ID)).thenReturn(details);
+            PowerMockito.when(org.smartregister.anc.library.AncLibrary.getInstance()).thenReturn(ancLibrary);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.smartregister.anc.library.AncLibrary.getInstance' due to existing import 'org.smartregister.anc.library.AncLibrary'](https://app.codacy.com/app/OpenSRP/opensrp-client-anc/pullRequest?prid=4499997)